### PR TITLE
Daily build workflow to publish snapshots and releases

### PR DESCRIPTION
**Pull request by mngan**
- Brought over the full build workflow from netbox-docker. 
- peering-manager/docker repo needs to have secrets (`dockerhub_username` and `dockerhub_password`) configured for this to work.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -1,35 +1,91 @@
...
   echo ""
   echo "Examples:"
   echo "  ${0} master"
   echo "              This will fetch the latest 'master' branch, build a Docker Image and tag it"
-  echo "              'peering-manager/peering-manager:latest'."
-  echo "  ${0} v1.2.0"
-  echo "              This will fetch the 'v1.2.0' tag, build a Docker Image and tag it"
-  echo "              'peeringmanager/peering-manager:v1.2.0' and 'peeringmanager/peering-manager:v1.2'."
-  echo ""
+  echo "              'respawner/peering-manager:latest'."
```
Comment (gmazoyer) [inline build.sh]: Old repository name used. Should now be `peering-manager/peering-manager`.

File: build.sh [hunk 2]
```diff
@@ -1,35 +1,91 @@
...
+  echo "              This will fetch the latest 'develop' branch, build a Docker Image and tag it"
+  echo "              'respawner/peering-manager:snapshot'."
+  echo "  ${0} v2.6.6"
+  echo "              This will fetch the 'v2.6.6' tag, build a Docker Image and tag it"
+  echo "              'respawner/peering-manager:v2.6.6' and 'respawner/peering-manager:v2.6'."
+  echo "  ${0} develop-2.7"
+  echo "              This will fetch the 'develop-2.7' branch, build a Docker Image and tag it"
+  echo "              'respawner/peering-manager:develop-2.7'."
+  echo "  SRC_ORG=respawner ${0} feature-x"
+  echo "              This will fetch the 'feature-x' branch from https://github.com/respawner/peering-manager.git,"
```
Comment (gmazoyer) [inline build.sh]: Old repository URL used. Should now be `https://github.com/peering-manager/peering-manager.git`.

**Review by gmazoyer — CHANGES_REQUESTED** [review]
Few details to change otherwise LGTM.

File: build.sh (hunk 2 above)
Comment (mngan) [reply build.sh]: Thanks for catching that, I thought I had gone through and changed those, but perhaps not. PR should be updated now.

**Review by gmazoyer — APPROVED** [review]

**Pull request merged**